include/windows/osd.h

```diff
@@ -682,6 +682,15 @@ static inline char* strsep(char **stringp, const char *delim)
 
 #define __attribute__(x)
 
+static inline int ofi_mmap_anon_pages(void **memptr, size_t size)
+{
+	return -FI_ENOSYS;
+}
+
+static inline void ofi_unmap_anon_pages(void *memptr, size_t size)
+{
```

Comment:
Why isn't this returning `-FI_ENOSYS`?

Reply:
I don't think there is anything we can do on failure to unmap a buffer, and we should never call this because we'll never succeed in allocating a buffer with the MMAP flag set.

Reply:
Changed to return errno to unify this with hugepage unmap.